packages/design-system/src/components/web-components/ds-dropdown.stories.tsx

```diff
@@ -0,0 +1,148 @@
+        'Sets the initial selected state. Using this attribute instead of `value` means that the select will maintain its own internal selection state.',
+      control: 'text',
+    },
+    disabled: { description: 'Disables the entire field.', control: 'boolean' },
+    'error-message': {
+      description: 'Enable the error state by providing an error message',
+      control: 'text',
+    },
+    'error-placement': {
+      description: 'Location of the error message relative to the field input',
+      options: [undefined, 'top', 'bottom'],
+      control: { type: 'radio' },
+    },
+    'field-class-name': {
+      description: 'Additional classes to be added to the dropdown button element',
+      control: 'text',
+    },
+    hint: {
+      description: 'Hint text or HTML',
+      control: 'text',
+    },
+    label: {
+      description: 'Label text or HTML.',
+      control: 'text',
+    },
+    'label-class-name': {
+      description: 'Additional classes to be added to the field label',
+      control: 'text',
+    },
+    'label-id': {
+      description:
+        "A unique `id` to be used on the field label. If one isn't provided, a unique ID will be generated.",
+      control: 'text',
+    },
+    name: {
+      description:
+        'The `name` is applied to a hidden select element that holds the selected value for the purpose of native HTML form support',
+      control: 'text',
+    },
+    options: {
+      description:
+        'The list of options to be rendered as an array of objects. If it is defined as an attribute in HTML, it needs to be stringified. Each item must have a `label` and `value`.',
+      control: 'object',
+    },
+    'requirement-label': { control: 'text' },
+    size: {
+      description: 'Sets the max-width of the input either to `"small"` or `"medium"`',
+      options: [undefined, 'medium', 'small'],
+      control: { type: 'radio' },
+    },
+    value: {
+      description:
+        'Sets the selected value. Using this attribute instead of `default-value` means the dropdown will not maintain its own internal selection state.',
+      control: 'text',
+    },
+  },
+  args: {
+    label: 'Dropdown example',
+    name: 'dropdown_field',
+    options,
+  },
+  parameters: {
+    docs: {
+      page: WebComponentDocTemplate,
+      description: {
+        component:
+          'For information about how and when to use this component, [refer to its full documentation page](https://design.cms.gov/components/dropdown/).',
+      },
+      componentEvents: {
+        'ds-change': {
+          description: 'Dispatched whenever the selected value changes.',
+          eventObjectDescription: (
+            <>
+              <code>event.details.target.value</code> - The <code>value</code> of the selected
+              option
+            </>
+          ),
+        },
+        'ds-blur': {
+          description: 'Dispatched whenever the dropdown loses focus.',
+        },
+      },
+    },
+  },
+};
+
+/*
```

Comment:
why is this commented out?

Reply:
I left it in there for a future state where we can use `<option>` children to define the options. I can add a TODO comment to it.